docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
     environment:
       # Django settings overrides:
       - ACCOUNT_DEFAULT_HTTP_PROTOCOL=http
-      - ALLOWED_HOSTS=localhost,localhost:8000
+      - ALLOWED_HOSTS=localhost,127.0.0.1,[::1],web,api
```

Comment:
not currently, but it doesn't hurt either. If we enable the ``nginx`` container, we'll need it.